tests_integration/test_session_token.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+
+@pytest.mark.integration
+def test_use_session_token(runner, snowflake_session):
+    session_token = snowflake_session.rest.token
+    master_token = snowflake_session.rest.master_token
+
+    result_of_setting_variable = runner.invoke(
+        [
+            "sql",
+            "-q",
+            "set a = 42",
+            "-x",
+            "--account",
+            snowflake_session.account,
+            "--session-token",
+            session_token,
+            "--master-token",
+            master_token,
+        ]
+    )
+    assert result_of_setting_variable.exit_code == 0
+    result_of_getting_variable = runner.invoke_json(
+        [
+            "sql",
+            "-q",
+            "select $a as dummy",
+            "-x",
+            "--account",
+            snowflake_session.account,
+            "--session-token",
+            session_token,
+            "--master-token",
+            master_token,
```

Comment:
Seeing the errors from integration tests I'm wondering if those parameters changes the session life time.  I'm guessing that existing tests start to fail after this one. To check it you can run `pytest -p no:randomly` to avoid random order of tests.

Reply:
Thanks, I'll review that.

Reply:
I have checked this and it seems like it was closing the session after invoking the CLI with the session token and master token. I have changed the code to set connection parameter `server_session_keep_alive` to `True` when the session token and master token are used, since in those cases, the process that creates the connection (and shares it) should be the one responsible for deleting the session when it is no longer used.